lib/codegen/fromcto/rust/rustvisitor.js

```diff
@@ -293,7 +348,9 @@ class RustVisitor {
         case 'Boolean':
             return 'bool';
         case 'Long':
-            return 'u64';
+            return 'i64';
```

Comment:
These changes basically follows what we do in [Go](https://github.com/accordproject/concerto-codegen/blob/72eb13b727f9c29a1250ee37d9fe0bf904832886/lib/codegen/fromcto/golang/golangvisitor.js#L286-L291), [Java](https://github.com/accordproject/concerto-codegen/blob/72eb13b727f9c29a1250ee37d9fe0bf904832886/lib/codegen/fromcto/java/javavisitor.js#L371-L376), and [C#](https://github.com/accordproject/concerto-codegen/blob/72eb13b727f9c29a1250ee37d9fe0bf904832886/lib/codegen/fromcto/csharp/csharpvisitor.js#L579-L584) and the corresponding [primitive types in Java](https://docs.oracle.com/javase/tutorial/java/nutsandbolts/datatypes.html).

I'm not sure if this will count as breaking change (probably they do) and we need to up a major version just for that.

Reply:
This is a bug fix, IMO. The concerto spec says that Long values are signed, and previously these values are unsighed.

We also have the long-standing issue around the representation of Longs in JavaScript. https://github.com/accordproject/concerto/issues/189